.github/workflows/deploy.yml

```
@@ -111,7 +111,7 @@
           ./andromeda-deploy
 
   trigger-schema-parser:
-      needs: [deploy, build_schemas]
+      needs: [deploy]
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)